### PR TITLE
Use constants in IdExtensions.IsSnowflake()

### LINKED ISSUE #8: Remove magic numbers in IdExtensions.IsSnowflake()
These lines in `IdExtensions.IsSnowflake()` contains constants that we can derive from the `Id` class:
            
            long timestamp = id >> 22;
            long thread = (id >> 17) & 0b11111;
            long process = (id >> 12) & 0b11111;
            long increment = id & 0b111111111111;
            
The numbers 22, 17 and 12 are just the sum of the (private) constants `xxxBits`, and the bitmasks are immediately available from the Id class.

----

I propose changes to the lines so that they use descriptive names:

            long timestamp = id >> TimestampOffset;
            long thread = (id >> ThreadOffset) & Id.ThreadIdMask;
            long process = (id >> ProcessOffset) & Id.ProcessIdMask;
            long increment = id & Id.IncrementMask;

I have a local branch with the required changes, and it has passed the unit tests.
If this idea works with you, I can create a pull request.

### PR DESCRIPTION
**Pull request by haruki-taka8**
Closes #8

### CODE REVIEW

File: src/FlakeId/Extensions/IdExtensions.cs [hunk 1]
```diff
@@ -8,14 +8,19 @@ public static class IdExtensions
         public static DateTimeOffset ToDateTimeOffset(this Id id) =>
             DateTimeOffset.FromUnixTimeMilliseconds(id.ToUnixTimeMilliseconds());
 
+        private const int TimestampOffset = Id.IncrementBits + Id.ProcessIdBits + Id.ThreadIdBits;
+        private const int ThreadOffset = Id.IncrementBits + Id.ProcessIdBits;
+        private const int ProcessOffset = Id.IncrementBits;
+
+
```
Comment (aevitas) [inline src/FlakeId/Extensions/IdExtensions.cs:15]: nit: unnecessary newline

**Review by aevitas — CHANGES_REQUESTED** [review]
Could you fix the extra newline and merge in the `master` branch? I've added a simple CI build so we should see green tests in the PR as well.

Thanks a lot for submitting this!

File: src/FlakeId/Extensions/IdExtensions.cs (hunk 1 above)
Comment (haruki-taka8) [reply src/FlakeId/Extensions/IdExtensions.cs]: Thanks! I didn't catch the extra empty line on my laptop.
I fixed it in da70313.

**Review by aevitas — APPROVED** [review]
LGTM, thank you for contributing. 👍

**Pull request merged**